run_Wochenende_SLURM.sh

```diff
@@ -62,10 +62,16 @@ cpus=12
 
 
 
+# 2020_09 massive reference
+python3 run_Wochenende.py --metagenome 2020_09_massiveref_human --threads $cpus --aligner bwamem --no_abra        --remove_mismatching 3 --readType SE --debug --force_restart $fastq
```

Comment:
looks fine, please delete space after --no_abra

Reply:
The space was added on purpose to make clear there is no --mq30 filter .